src/main/java/org/radarcns/management/ManagementPortalTestApp.java

```diff
@@ -0,0 +1,88 @@
+package org.radarcns.management;
+
+import io.github.jhipster.config.JHipsterConstants;
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import java.util.Arrays;
+import java.util.Collection;
+import javax.annotation.PostConstruct;
+import org.radarcns.management.config.ApplicationProperties;
+import org.radarcns.management.config.DefaultProfileUtil;
+import org.radarcns.management.config.ManagementPortalProperties;
+import org.radarcns.management.config.SourceTypeLoader;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.actuate.autoconfigure.MetricFilterAutoConfiguration;
+import org.springframework.boot.actuate.autoconfigure.MetricRepositoryAutoConfiguration;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.autoconfigure.liquibase.LiquibaseProperties;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.cloud.netflix.zuul.EnableZuulProxy;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.FilterType;
+import org.springframework.core.env.Environment;
+
+@ComponentScan(excludeFilters = @ComponentScan.Filter(type = FilterType.ASSIGNABLE_TYPE, value = SourceTypeLoader.class))
+@EnableZuulProxy
+@EnableAutoConfiguration(exclude = {MetricFilterAutoConfiguration.class, MetricRepositoryAutoConfiguration.class})
+@EnableConfigurationProperties({LiquibaseProperties.class, ApplicationProperties.class ,
+    ManagementPortalProperties.class})
+public class ManagementPortalTestApp {
```

Comment:
Perhaps it would be good to add a Javadoc comment to explain how this class is different from the regular `ManagementPortalApp` and why it is necessary.